<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>234</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>533</width>
    <height>234</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="Myslider" name="duration_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>67</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="song_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
   </widget>
   <widget class="QLabel" name="duration_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>84</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 12px;
	border-color: rgb(255, 102, 103);
	background-color: rgb(22, 22, 22);
	color: rgb(255, 255, 255);
    min-width: 80px;
font: 75 9pt &quot;Fira Code&quot;;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>84</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 12px;
	border-color: rgb(255, 102, 103);
	background-color: rgb(22, 22, 22);
	color: rgb(255, 255, 255);
    min-width: 80px;
font: 75 9pt &quot;Fira Code&quot;;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>84</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 12px;
	border-color: rgb(255, 102, 103);
	background-color: rgb(22, 22, 22);
	color: rgb(255, 255, 255);
    min-width: 80px;
font: 75 9pt &quot;Fira Code&quot;;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mute_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="random_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDial" name="volumen_dial">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="sliderPosition">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background: none;
font: 75 9pt &quot;Fira Code&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background: none;
font: 75 9pt &quot;Fira Code&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="volumen_LCD">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>139</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;
background: none;
color: rgb(85, 255, 0);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>160</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background: none;
font: 75 9pt &quot;Fira Code&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>117</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background: none;
font: 75 9pt &quot;Fira Code&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Song List</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-250</x>
      <y>-120</y>
      <width>1248</width>
      <height>473</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resources/minimalistic dragon hd wallpaper black.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>duration_slider</zorder>
   <zorder>song_list</zorder>
   <zorder>duration_label</zorder>
   <zorder>pause_button</zorder>
   <zorder>play_button</zorder>
   <zorder>stop_button</zorder>
   <zorder>mute_button</zorder>
   <zorder>random_button</zorder>
   <zorder>volumen_dial</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>volumen_LCD</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="file_open"/>
    <addaction name="separator"/>
    <addaction name="file_exit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionDelete_Song"/>
    <addaction name="actionDelete_All"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="file_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDelete_Song">
   <property name="text">
    <string>Delete Song</string>
   </property>
  </action>
  <action name="actionDelete_All">
   <property name="text">
    <string>Delete All</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Myslider</class>
   <extends>QSlider</extends>
   <header location="global">myslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
